### DISCUSSION TITLE
Filters should be Macroable

### DISCUSSION THREAD

**Discussion by tylernathanreed** [laravel/nova-issues#6429] [Ideas & Feature Requests]
I've been working on expanding Nova functionality for my project, and I noticed that Filters aren't macroable. Short of creating custom classes for every single filter, or repeating logic in a grotesque manner, I'm at a loss.

Please add the Macroable trait to `Laravel\Nova\Filters\Filter`.

**Comment by crynobone** [comment]
Released with Laravel Nova 5.0.0

Feel free to open up a new issue if you're still experiencing this problem on the latest version.
